lib/compeon/api_client.rb

```diff
@@ -6,7 +6,8 @@
 
 module Compeon
   class APIClient
-    def initialize(version:, token: nil)
+    def initialize(version:, token: nil, url: ENV['COMPEON_API_URL'])
+      @url = url
```

Comment:
What about making the version a part of the url, as it otherwise has no purpose?

Reply:
In this way you can use different API version with different client instances.

Reply:
You can do that too.
```ruby
Compeon::APIClient.new(token: 'token', url: 'https://api.compeon.de/v1')
```

Reply:
But then you end up in.
```ruby
Compeon::APIClient.new(url: "#{ENV['COMPEON_API_URL']}/v1')
Compeon::APIClient.new(url: "#{ENV['COMPEON_API_URL']}/v2')
```
instead of
```ruby
Compeon::APIClient.new(version: 1)
Compeon::APIClient.new(version: 2)
```

Reply:
Maybe we could even default the API version.

Reply:
There's ups and downs for both approaches. Was just a suggestion because the version is not used in any other way except than building the URL and it might imply different behavior for different versions.

Reply:
I don't see any benefit in `url: "#{ENV['COMPEON_API_URL']}/v1"`.

> it might imply different behavior for different versions

Yes, that's why I used an explicit arg. So the client can change for example pagination settings or thinks like that depending on the version.